ssl/statem/statem_srvr.c

```diff
@@ -3229,8 +3229,9 @@ static int tls_process_cke_gost(SSL_CONNECTION *s, PACKET *pkt)
         return 0;
     }
     if (EVP_PKEY_decrypt_init(pkey_ctx) <= 0) {
+        EVP_PKEY_CTX_free(pkey_ctx);
```

Comment:
This now isn't necessary since the err block already does it.

Reply:
okay thank you